tests/cbmc/proofs/s2n_stuffer_write_reservation/Makefile

```diff
@@ -38,16 +38,16 @@ PROJECT_SOURCES += $(SRCDIR)/utils/s2n_safety.c
 
 # We abstract these functions because manual inspection demonstrates they are unreachable.
 REMOVE_FUNCTION_BODY += s2n_blob_slice
+REMOVE_FUNCTION_BODY += s2n_stuffer_wipe
 REMOVE_FUNCTION_BODY += s2n_stuffer_wipe_n
 
-UNWINDSET += s2n_stuffer_write_network_order.0:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.1:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.2:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.3:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.4:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.5:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.6:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_network_order.7:$(shell echo $$((1 + $(MAX_LENGTH))))
-UNWINDSET += s2n_stuffer_write_reservation_harness.0:$(shell echo $$((1 + $(MAX_LENGTH))))
+UNWINDSET += s2n_stuffer_write_network_order.0:9
+UNWINDSET += s2n_stuffer_write_network_order.1:9
+UNWINDSET += s2n_stuffer_write_network_order.2:9
+UNWINDSET += s2n_stuffer_write_network_order.3:9
+UNWINDSET += s2n_stuffer_write_network_order.4:9
+UNWINDSET += s2n_stuffer_write_network_order.5:9
+UNWINDSET += s2n_stuffer_write_network_order.6:9
+UNWINDSET += s2n_stuffer_write_network_order.7:9
```

Comment:
Do you really want to hard-code constants here?  At least make them a makefile variable?

Reply:
Implemented as suggested.